README.md

```diff
@@ -1,17 +1,169 @@
+architecture. Check the official [documentation](https://graphdb.ontotext.com/documentation/10.4/aws-deployment.html)
+for more details.
 
-Before you begin using this Terraform module, ensure you meet the following prerequisites:
+## Table of contents
+
+- [About GraphDB](#about-graphdb)
+- [Features](#features)
+- [Versioning](#versioning)
+- [Prerequisites](#prerequisites)
+- [Inputs](#inputs)
+- [Usage](#usage)
+- [Examples](#examples)
+- [Local Development](#local-development)
+- [Release History](#release-history)
+- [Contributing](#contributing)
+- [License](#license)
+
+## About GraphDB
+
+<p>
+  <a href="https://www.ontotext.com/products/graphdb/">
+    <picture>
+      <img src="https://www.ontotext.com/wp-content/uploads/2022/09/Logo-GraphDB.svg" alt="GraphDB logo" title="GraphDB" height="75">
+    </picture>
+  </a>
+</p>
+
+Ontotext GraphDB is a highly efficient, scalable and robust graph database with RDF and SPARQL support. With excellent
+enterprise features,
+integration with external search applications, compatibility with industry standards, and both community and commercial
+support, GraphDB is the
+preferred database choice of both small independent developers and big enterprises.
+
+## Features
+
+The module provides the building blocks of configuring, deploying and provisioning a highly available cluster of GraphDB
+across multiple availability zones using EC2 Autoscaling Group. Key features of the module include:
+
+- EC2 Autoscaling Group
+
+- Network Load Balancer
+
+- NAT Gateway for outbound connections
+
+- Route53 Private Hosted Zone for internal GraphDB cluster communication
+
+- IAM Policies and roles
+
+- VPC
+
+- Monitoring
 
-- **Terraform Installed**: You should have Terraform installed on your local machine. You can download Terraform from the [official website](https://www.terraform.io/downloads.html).
+- Backup
 
-- **AWS Credentials**: Ensure that you have AWS credentials configured on your machine. You can configure AWS access keys and secret keys using the [AWS CLI](https://aws.amazon.com/cli/).
+- and many more
 
-- **Terraform AWS Provider Configuration**: Configure the AWS provider in your Terraform project. You can add your AWS access and secret keys as environment variables or use other methods for provider configuration.
+## Versioning
 
-- **Terraform Configuration File**: Create a new Terraform configuration file in your project, and use this module by specifying its source.
+The Terraform module follows the Semantic Versioning 2.0.0 rules and has a release lifecycle separate from the GraphDB
+versions. The next table shows the version compatability between GraphDB and the Terraform module.
+
+| GraphDB Terraform | GraphDB        |
+|-------------------|----------------|
+| Version 0.0.1     | Version 10.6.x |
+
+You can track the particular version updates of GraphDB in the [changelog](CHANGELOG.md) or the [release notes](https://github.com/Ontotext-AD/terraform-aws-graphdb/releases).
+
```

Comment:
Please add a reference to the graphdb release notes. https://graphdb.ontotext.com/documentation/10.6/release-notes.html 

Could be as a link (Version 10.6.x) in the table.